README.md

```diff
@@ -9,26 +9,20 @@ See https://www.consul.io/docs/ecs for full documentation.
 
 ## Architecture
 
-![Architecture](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/_docs/architecture.png?raw=true)
+![Architecture](./_docs/architecture.png?raw=true)
 
 Each task is created via the `mesh-task` module. This module adds
 additional containers known as sidecar containers to your task definition.
 
 Specifically, it adds the following containers:
 
-* `consul-ecs-mesh-init` – Runs at startup to set up initial configuration for Consul and Envoy.
-* `consul-client` – Runs for the full lifecycle of the task. This container runs a
-  [Consul client](https://www.consul.io/docs/architecture) that connects with
-  Consul servers and configures the sidecar proxy.
-* `sidecar-proxy` – Runs for the full lifecycle of the task. This container runs
-  [Envoy](https://www.envoyproxy.io/) which is used to proxy and control
-  service mesh traffic. All requests to and from the application run through
-  the sidecar proxy.
-* `health-sync` - Runs for the full lifecycle of the task. This container
-  syncs health check statuses from ECS into Consul.
+* `consul-ecs-control-plane` – Runs for the full lifecycle of the task. At startup, it connects to the available Consul servers and performs a login with the configured IAM Auth method and obtains an ACL token with appropriate privileges. Using the token, it registers the service and proxy entities to Consul's catalog. It then bootstraps the configuration JSON required by the Consul dataplane container and writes it to a shared volume. After this, the container enters into its reconciliation loop where it periodically syncs the health of ECS containers into Consul. Upon receiving SIGTERM, it marks the corresponding service instance in Consul as unhealthy and waits for the dataplane container to shutdown after which it deregisters the service and proxy entities from Consul's catalog and performs a Consul logout.
```

Comment:
I found this block of text kind of hard to grok. What do you think about using sub-bullets to organize it?

```suggestion
* `consul-ecs-control-plane` – Runs for the full lifecycle of the task.
  * At startup it connects to the available Consul servers and performs a login with the configured IAM Auth method to obtain an ACL token with appropriate privileges.
  * Using the token, it registers the service and proxy entities to Consul's catalog.
  * It then bootstraps the configuration JSON required by the Consul dataplane container and writes it to a shared volume.
  * After this, the container enters into its reconciliation loop where it periodically syncs the health of ECS containers into Consul.
  * Upon receiving SIGTERM, it marks the corresponding service instance in Consul as unhealthy and waits for the dataplane container to shutdown.
  * Finally, it deregisters the service and proxy entities from Consul's catalog and performs a Consul logout.
```